### ISSUE TITLE
Upload to pypi

### ISSUE THREAD

**Issue by thorsummoner** [erkin/ponysay#264]
I mean, after the whole "replace/rebuild the build system" and "use the standard python packaging toolchain", I would love to install this from the toolchain standard database as well <3 Cool project, I too am a nerd.

Maybe if it helps, I also left this comment: https://github.com/erkin/ponysay/issues/248#issuecomment-317531927

**Comment by jristz** [comment]
I agree that something could be done to fix the build system #94 and I will agree with anything that does the work...

Also pip will help a little to me on some tasks.

**Comment by loxaxs** [comment]
#248 #264 #273
